### ISSUE TITLE
TypeError: 'NoneType' object is not iterable

### ISSUE THREAD

**Issue by foranastes** [vaibhavsingh97/random-word#67]
**Describe the bug**
I'm getting TypeError: 'NoneType' object is not iterable when I try to convert words to string.

**To Reproduce**
```
def generate_randome():
    r = RandomWords()
    the_list = list(r.get_random_words(limit=10))
    randome_words = ' '.join(the_list)
    return randome_words
```

**Comment by illyanyc** [comment]
I have the same issue, looks like `r.get_random_words()` sometimes returns an empty list

**Comment by arixank** [comment]
yes it does, return an empty word even on r.get_random_word()..!

**Comment by foranastes** [comment]
I think it's a problem with the API used to generate words. Do they've any limits or something?

**Comment by illyanyc** [comment]
passing the following params into RandomWords solved the problem, works fine in production.

I think including `hasDictionaryDef` and `includePartsOfSpeech` is necessary.

`r = RandomWords()
mnemonic_list = r.get_random_words(
                              hasDictionaryDef="true", 
                              includePartOfSpeech="noun,verb", 
                              minLength=4, 
                              maxLength=6, 
                              sortBy="alpha", 
                              sortOrder="asc", 
                              limit=24)`
